package.json

```diff
@@ -3,7 +3,7 @@
   "version": "5.0.0",
   "description": "Returns an array of all tabbable DOM nodes within a containing node.",
   "main": "dist/index.min.js",
```

Comment:
Shouldn't we also not reference the minified version of the CJS bundle for the same reasons?

Reply:
I think we should reference the non-minified version of the CJS bundle for the same reasons.